exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/Blockchain.java

```diff
@@ -55,11 +56,26 @@ public static Blockchain newInstance(View view) {
   }
 
   /**
-   * Returns the height of the latest committed block in the blockchain.
-   * The height can be considered as a count of blocks in the blockchain,
-   * where the first block has height {@code h = 0}. For example,
-   * the "genesis block" (first, initial block in the blockchain) has height {@code h = 0}.
-   * The latest committed block has height {@code h = getAllBlockHashes().size() - 1}.
+   * Returns true if the blockchain contains <em>exactly</em> the same block as the passed
+   * value; false if it does not contain such block. Please note that all block fields
+   * are compared, not only its hash.
+   *
+   * @param block a value to check for presence in the blockchain
+   */
+  public boolean containsBlock(Block block) {
+    return findBlock(block.getBlockHash())
+        .map(block::equals)
+        .orElse(false);
+  }
+
+  /**
+   * Returns the height of the latest committed block in the blockchain,
+   * or <em>blockchain height</em>. The height is a distance between the last block
```

Comment:
```suggestion
   * in the other words <em>blockchain height</em>. The height is a distance between the last block
```

Reply:
Or the other way around: returns the blockchain height, which is the height of the latest committed block in the blockchain?